plugin/samples/authorized-type.yml

```diff
@@ -0,0 +1,42 @@
+---
+kind: pipeline
```

Comment:
There are different `kind`s in drone, [such as secrets](https://docs.drone.io/secret/encrypted/), which should not be processed as if they are a `kind: pipeline`. So the default policy likely needs some tweaking to properly understand which YAML documents to process.

Reply:
Nice, updated the policy to deny based on `pipeline` and `kind`.